src/templates/data-edit.html

```diff
@@ -504,6 +509,7 @@ <h2>Charity Categories</h2>
 			currency_id: parseInt(ui.countryCurrency.value, 10),
 			min_donation_amount: parseInt(ui.countryMinDonationAmount.value, 10),
 			min_donation_currency_id: parseInt(ui.countryMinDonationCurrencyId.value, 10),
+			gift_aid: parseFloat(ui.giftAid.value)
```

Comment:
can you add a trailing comma, please?
```suggestion
			gift_aid: parseFloat(ui.giftAid.value),
```